model/riscv_vlen.sail

```diff
@@ -0,0 +1,37 @@
+type elen : Int = 64
+
+register VLEN : bits(2)
+
+val get_vlen : unit -> {|64, 128, 256, 512|}
```

Comment:
There are V-spec implementations with VLEN 16,384 and SMD supports at least up to 2048. Not sure where we should put the limit, 16K is a bit on the extreme side.

Reply:
The VLEN domain is now extended to 32 ~ 16K.